code/game/objects/effects/temporary_visuals/miscellaneous.dm

```diff
@@ -403,29 +403,6 @@
 	pixel_y = rand(8,12)
 	animate(src, pixel_y = pixel_y + 16, alpha = 0, time = duration)
 
-/obj/effect/temp_visual/love_heart
```

Comment:
Aren't these hearts generated when you hug things, like Ian?

Reply:
No, these are way fatter and show over your body, not your head.